### PR TITLE
[CHORE] Drop support for Ember 1.11 and 1.12

### PR DESCRIPTION
**Pull request by cyril-sf**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,16 +26,13 @@ env:
     # - EMBER_TRY_SCENARIO=ember-release
     # - EMBER_TRY_SCENARIO=ember-beta
     # - EMBER_TRY_SCENARIO=ember-canary
-    - EMBER_TRY_SCENARIO=ember-1.11
-    - EMBER_TRY_SCENARIO=ember-1.12
     - EMBER_TRY_SCENARIO=ember-1.13
     - EMBER_TRY_SCENARIO=ember-2.0
-    - EMBER_TRY_SCENARIO=ember-default
+    # - EMBER_TRY_SCENARIO=ember-default
```
Comment (mixonic) [inline .travis.yml]: Why this? Testing the default setup seems quite valid?

Comment (cyril-sf) [reply .travis.yml]: I may have been overzealous.

I was trying to limit the work done by Travis that we may not care about.

Comment (mixonic) [reply .travis.yml]: I'd like to see it come back. The default dependencies should definitely pass, or out of the box when you clone the repo the suite may be red.

```suggestion
    - EMBER_TRY_SCENARIO=ember-default
```

Comment (cyril-sf) [reply .travis.yml]: I didn't mean to not bring it back, I was dealing with another project

**Pull request merged**